documents/token.md

```diff
@@ -0,0 +1,190 @@
+## Tokens
+
+### Fetch token by payment id
```

Comment:
@HancilSequeira don’t we have create a token API? Or am I missing something?

Reply:
There is no API mentioned for this also its is not mentioned [here](https://docs.google.com/spreadsheets/d/13T4KAv5VydqWWDLDtQwkCczPPpFKHvvZORgyWAcYWQg/edit#gid=1152367028)